### ISSUE TITLE
For the card count on split piles, could you show something like 5/5

### ISSUE THREAD

**Issue by wvoigt** [sumpfork/dominiontabs#107]
Suggested by @soudek01 in #73

**Comment by soudek01** [comment]
Suggestions:

Do this for all piles with multiple cards:

Hermit/Madman: 10/10
Urchin/Mercenary: 10/10
Page->Champion: 10/5/5/5/5
Peasant->Teacher: 10/5/5/5/5
Tournament and Prizes: 10/1/1/1/1/1
Castles: 2/1/2/1/2/1/1/2
Split Piles: 5/5

**Comment by soudek01** [comment]
Shelters 6/6/6
Knights: 1/1/1/1/1/1/1/1/1/1

(I am unsure about the Castles and Knights, they may be fine as they are).

**Comment by wvoigt** [comment]
I'm a little worried about how long these strings can be.  The line they are placed can also have images (and in the future the card types).  So things could get crowded.  How about something like:

- Hermit/Madman: 2x[10]
- Urchin/Mercenary: 2x[10]
- Page->Champion: [10] 4x[5]
- Peasant->Teacher: [10] 4x[5]
- Tournament and Prizes: [10] 5x[1]
- Castles: 3x[2] 4x[1]
- Split Piles: 2x[5]
- Shelters 3x[6]
- Knights: 10x[1]

where [ ] is the card image

**Comment by soudek01** [comment]
This is an awesome idea.

**Comment by wvoigt** [comment]
The brackets represent the card image that is being used.  
![type-cost](https://cloud.githubusercontent.com/assets/1182468/20469618/d2700dfc-af66-11e6-9c14-168a021390b5.jpg)

The changes for this have been added to Pull #105

**Comment by soudek01** [comment]
That looks great to me! I think you were considering underlining card names?

**Comment by wvoigt** [comment]
@soudek01 yes, that is the case now in the branch [Card-DB-refactoring](https://github.com/sumpfork/dominiontabs/tree/Card-DB-refactoring).  I took the image before I had gone through and added all the style changes. 

But note, that I spent my time on en_us.  The other languages will not all have the underlines if the cards have been translated.  
If they have not been translated (as is often the case with the two above), then the translation will inherit the English version until someone translates the card (so it will have the underlines.)
